### ISSUE TITLE
How are ideas licensed?

### ISSUE THREAD

**Issue by aschrijver** [samsquire/ideas#42]
Thank you for this inspiring repository. I see people are contributing to it with their input, but since there is no licensing all of the ideas mentioned are proprietary, all rights reserved. It is fine if that is your intention, but you may make that more clear. Even better would be if you'd consider giving the repo e.g. one of the Creative Commons licenses (like CC-BY-NC-SA).

**Comment by karstenwinter** [comment]
I don't think you can license ideas. If you're afraid it would be used without your Consent you wouldn't post it here Publicly or would have Patented it beforehand if it was a Million dollar idea

Might be wrong but in game dev ideas are plenty and good Implementation super rare ;)

**Comment by aschrijver** [comment]
Ah, I mean not the concept of the idea itself, but the text you've written. I wanted to quote idea 85 with attribution and a link to the original location. In the end I decided to leave out the quote, but this repo can disappear at any time.

**Comment by karstenwinter** [comment]
Ah, thanks for clarification

They have permalinks, so you could, of course you're right that this may not be a stable link
https://github.com/samsquire/ideas#85-software-metadata

if you want to make super sure your link will last as long as the internet you can use wayback machine
https://web.archive.org/web/20210703232821/https://github.com/samsquire/ideas#85-software-metadata

**Comment by aschrijver** [comment]
Yes, but I think this does not solve it, because now archive.org offers "Copyright Sam Squire, All rights reserved" proprietary data.

**Comment by karstenwinter** [comment]
fork the repo :) Then it will not disappear :)

**Comment by samsquire** [comment]
Hi aschrijver karstenwinter

Sorry it has taken so long to reply.

I've decided to licence it under Creative Commons Share alike licence. I need to change the licence on the ideas pages themselves.

<a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons Licence" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
